MODULE.bazel

```diff
@@ -40,6 +41,12 @@ bazel_dep(name = "sqlite3", version = "3.42.0.bcr.1")  # C code for SQLite.
 bazel_dep(name = "toolchains_llvm", version = "1.1.2")
 bazel_dep(name = "toolchains_protoc", version = "0.3.6")
 
+git_override(
```

Comment:
There are no releases so we must use `git_override`? Add a comment explaining that?

Reply:
Yeah I think it's at a point where it could go to bCR now, will leave myself a TODO